src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -174,7 +175,8 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
                         subset_dir='pixel_mat_subsetted',
                         norm_vals_name='channel_norm_post_rowsum.feather', is_mibitiff=False,
                         blur_factor=2, subset_proportion=0.1, seed=42,
-                        channel_percentile=0.99, multiprocess=False, batch_size=5):
+                        channel_percentile=0.99, channel_percentile_postnorm=0.999,
```

Comment:
Sorry to be super annoying about this, but could we remove `channel_percentile` for now (and add it back in when we add back in the first normalization step)? Since `channel_percentile` isn't being used now, it's a little confusing to have it still in the function definition.